config/settings.yml

```diff
@@ -1023,6 +1023,7 @@
   :interval: 60
   :memcache_server: 127.0.0.1:11211
   :memcache_server_opts: "-l 127.0.0.1"
```

Comment:
@jrafanie I just re-checked with only setting memcache_server_opts and the session update failed in this case issuing just the WARN log line but fails to update the session with the relevant data. - The dalli implementation looks to me like it is completely ignoring the store request for the session:

```ruby
    def guard_max_value(key, value)
      if value.bytesize <= @options[:value_max_bytes]
        yield
      else
        message = "Value for #{key} over max size: #{@options[:value_max_bytes]} <= #{value.bytesize}"
        raise Dalli::ValueOverMaxSize, message if @options[:error_when_over_max_size]

        Dalli.logger.warn message
        false
      end
    end
```


In my case (editing of a group) did not work. Issue #10074 has been closed meanwhile. As a solution for that issue the constructed Tree the appliances tags is no longer stored in the users session context.

I am not sure if the memcached overflow can happen anymore. With the fix to #10074 I can at least no longer find a way to reproduce the effects in manageiq.